### ISSUE TITLE
[Bug]: Reader view endlessly reloads recent posts

### ISSUE THREAD

**Issue by markbiek** [Automattic/wp-calypso#62520]
### Quick summary

When I go to https://wordpress.com/read, it just continuously reloads my recent posts. You can see the UI refreshing and all of the ajax calls in the dev console.

I'm assuming it's something unique to my account and probably related to all of the test sites I have.

https://user-images.githubusercontent.com/917632/161784632-2aaab6cf-5496-4784-8a45-b1bf696de643.mp4

### Steps to reproduce

1. Go to https://wordpress.com/read as the `markbiek` user

### What you expected to happen

I expected the list of recent posts to load once.

### What actually happened

It loads over and over.

### Browser

Google Chrome/Chromium, Mozilla Firefox

### Reproducibility

Consistent

### Severity

One

**Comment by markbiek** [comment]
There are two posts that it's cycling between in particular:

- https://public-api.wordpress.com/rest/v1.2/read/feed/128762176/posts/3925867239?http_envelope=1&content_width=720
- https://public-api.wordpress.com/rest/v1.2/read/feed/128762176/posts/3930590403?http_envelope=1&content_width=720

Both of which are coming from site `204776557` which is the site I've been trying to sure as a headless CMS for my personal blog.

**Comment by markbiek** [comment]
After more digging, this was just a weird glitch with one site. I had tried to setup the site at this domain once before and Jetpack got all out of whack. I had to have support disconnect Jetpack and the domain, then set everything up from scratch.

I think Calypso was just confused by the "Hello World" posts technically coming from two different sites even thought the domain & ID was the same.

Everything is working properly now.
